### PR TITLE
fixed "showing a weird pagination with filtering the minimum number o…

### PR DESCRIPTION
**Pull request by Hmone23**
@biplobice 

> I have added a C5J Rating Page List block with filtering the minimum number of ratings with 1.
> I have only one page with the rating 1, but it's showing a weird pagination with 3 pages!

The above error disappeared after clear cache, but I also checked it in the view.php.
Please confirm it.

### CODE REVIEW

**Comment by biplobice** [comment]
@Hmone23 
What you have changed in the `view()` method?
It looks exactly the same as the parent method to me.

File: blocks/c5j_rating_page_list/controller.php [hunk 1]
```diff
@@ -338,4 +338,43 @@ public function isValidControllerTask($method, $parameters = []): bool
...
+            }
+        }
+        
+        //Pagination...
+        $showPagination = false;
+        if ($this->num > 0) {
+            $list->setItemsPerPage($this->num);
+            $pagination = $list->getPagination();
+            $pages = $pagination->getCurrentPageResults();
+            if ($pagination->haveToPaginate() && $this->paginate && count($list->getResults()) === count($pagination)) {
```
Comment (Hmone23) [inline blocks/c5j_rating_page_list/controller.php:364]: @biplobice 

I checked here

**Comment by biplobice** [comment]
@Hmone23 
That's not the proper way to fix it.
Now, `getResult()` & `getTotalResults()` return different values.
You should fix it so that it returns the same results.

**Comment by biplobice** [comment]
Hmm, something weird is happening. Fixing that doesn't work.

**Comment by Hmone23** [comment]
@biplobice 

> Hmm, something weird is happening. Fixing that doesn't work.

It works fine in my local

https://user-images.githubusercontent.com/43093665/174025986-350aa283-cfb8-412c-9a47-096f214b9941.mov

**Comment by biplobice** [comment]
@Hmone23 
Did you fix the `getTotalResults()` method or saying about the current code?

**Comment by biplobice** [comment]
Fixed in https://github.com/concrete5cojp/c5j_ratings/pull/19

**Pull request closed without merging**